pkg/util/sys/device.go

```diff
@@ -481,3 +482,17 @@ func parseUdevInfo(output string) map[string]string {
 	}
 	return result
 }
+
+// ListDevicesChild list all child available on a device
+func ListDevicesChild(executor exec.Executor, device string) ([]string, error) {
+	cmd := "lsblk for child"
+	var childList []string
+
+	childListRaw, err := executor.ExecuteCommandWithOutput(false, cmd, "lsblk", "--noheadings", "--pairs", path.Join("/dev", device))
+	if err != nil {
+		return childList, fmt.Errorf("failed to list all devices: %+v", err)
```

Comment:
This will short circuit on an error. If this function is meant to return partial success, I would expect that it will continue parsing lines after seeing an error and return error and results at the end.

Reply:
What do you mean by partial success? I agree the error message must be improved but the `lsblk` command should return at least 1 line at this point.